test.js

```diff
@@ -2200,15 +2370,67 @@ exports.test = function (global) {
                       }
            }
         }
+      ],
+      satelliteGraphTests = [
+        {
+           name: "aql-traversal-index-join",
+           params: { func: genericSatelliteGraph,
+                      queryString: `
+                        FOR v, e, p IN 1..3 OUTBOUND CONCAT(@v, "/smart0:test0") GRAPH @g
+                          FOR doc in @@c
+                            FILTER doc.value1 == v.value1
+                            RETURN doc
+                      ` }
+        },
+        {
+           name: "aql-traversal-graph",
+           params: {  func: genericSatelliteGraph,
+                      queryString: `
+                        FOR v, e, p IN 1..3 OUTBOUND CONCAT(@v, "/smart0:test0") GRAPH @g
+                          return v`,
+                      bindParamModifier: function(param, bindParam) { delete bindParam["@c"]; }
+                    }
+        },
+        {
+           name: "aql-index-traversal-graph",
+           params: {  func: genericSatelliteGraph,
+                      queryString: `
+                        for doc in @@c
+                          filter doc.value1 >= 0 and doc.value1 <= 10
+                          let vkey = CONCAT(@v,"/smart", doc.value3, ":test", doc.value3)
+                          for v, e, p in 1..4 outbound vkey graph @g
+                            return {doc, p}
+                        `,
+                    }
+        },
```

Comment:
One additional test, where a `DistributeNode` would distribute the lookup-values to the DBServers, would be nice, so we can later see the improvement when e.g. the `remove-satellite-joins` rule is extended to `DistributeNode`s.